### PR TITLE
Add chickenshock dynamic time

### PR DESCRIPTION
**Pull request by marlau97**
Please check if cloning still works now. We didn't change it. @Aaron2000119 @maexled 

How to test: Create a chickenshock game with a time (default time is 50), test it by going to http://localhost/minigames/chickenshock#{id}

Closes Gamify-IT/issues#239

### CODE REVIEW

File: src/components/EditMinigameModals/EditChickenshockConfigurationModal.vue [hunk 1]
```diff
@@ -235,6 +243,11 @@ function addWrongAnswer() {
         >
           add question
         </b-button>
+        <a> Game Time (in seconds): </a>
+        <b-form-input
+          id="TimeInput"
+          v-model="configuration.time"
+        ></b-form-input>
```
Comment (maexled) [inline src/components/EditMinigameModals/EditChickenshockConfigurationModal.vue:289]:
```suggestion
        <b-form-group
        label-cols-lg="6"
        label="Game Time (in seconds)"
        label-for="input-sm"
      >
        <b-form-input
          id="time-input"
          type="number"
          v-model="configuration.time"
        />
      </b-form-group>
```
Make input type number and should show text and input text box in same line

**Review by maexled — COMMENTED** [review]
Also, I see we get a problem since we have a new attribute `time` in chickenshock configuration model. This would affect import/export. Are you okay if I fix this tomorrow in this branch?

File: src/components/EditMinigameModals/EditChickenshockConfigurationModal.vue [hunk 2]
```diff
@@ -12,6 +18,7 @@ import { useToast } from "vue-toastification";
 import { putMinigame } from "@/ts/rest-clients/minigame-rest-client";
 import { useRoute } from "vue-router";
 import { ITask } from "@/ts/models/overworld-models";
+import { BFormInput } from "bootstrap-vue-3";
```
[the remark this answers is not in the document]
Comment (marlau97) [reply src/components/EditMinigameModals/EditChickenshockConfigurationModal.vue]: applied

File: src/components/EditMinigameModals/EditChickenshockConfigurationModal.vue [hunk 3]
```diff
@@ -1,3 +1,9 @@
+<style scoped>
+#TimeInput {
+  width: 3vw;
+}
+</style>
```
[the remark this answers is not in the document]
Comment (marlau97) [reply src/components/EditMinigameModals/EditChickenshockConfigurationModal.vue]: applied

File: src/components/EditMinigameModals/EditChickenshockConfigurationModal.vue (hunk 1 above)
Comment (marlau97) [reply src/components/EditMinigameModals/EditChickenshockConfigurationModal.vue]: applied

**Review by maexled — APPROVED** [review]

**Review by Aaron2000119 — APPROVED** [review]

File: src/components/EditMinigameModals/EditChickenshockConfigurationModal.vue [hunk 4]
```diff
@@ -115,6 +122,7 @@ function resetModal() {
 }
 
 function handleOk() {
+  console.log("kekw awdawdawdadwdWDawd" + configuration.value.time);
```
[the remark this answers is not in the document]
Comment (marlau97) [reply src/components/EditMinigameModals/EditChickenshockConfigurationModal.vue]: not sure where this is supposed to be? shows outdated and I don't find it anywhere either, so I'm resolving this

**Pull request merged**